### PR TITLE
chore: add safe for failling streaming assets

### PR DESCRIPTION
**Pull request by dalkia**
## What does this PR change?

We had a recent crash due to StreamingAssets (aka Addressables) not being uploaded. This PR would check that the folder at least exists.

Here is the CI build for the failed streaming assets folder https://app.circleci.com/pipelines/github/decentraland/unity-renderer/15492/workflows/71796997-fb45-44fa-a9b2-5347cb00fa0c/jobs/87436

## How to test the changes?

Im making a push without building streaming assets. This should fail just for not having them

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/Editor/BuildCommand.cs [hunk 1]
```diff
@@ -141,12 +141,10 @@ static string getEnv(string key, bool secret = false, bool verbose = true)
 
     static void PerformBuild()
     {
-        Console.WriteLine(":: Building addressables");
+           Console.WriteLine(":: Building addressables");
 
         AddressableAssetSettings.CleanPlayerContent(
             AddressableAssetSettingsDefaultObject.Settings.ActivePlayerDataBuilder);
-        AddressableAssetSettings.BuildPlayerContent();
```
Comment (kuruk-mm) [inline unity-renderer/Assets/Scripts/Editor/BuildCommand.cs:148]: 🤔 this is correct?

Comment (dalkia) [reply unity-renderer/Assets/Scripts/Editor/BuildCommand.cs:148]: Im using it as assets. I dont want them to get build to confirm that the safe works.

File: browser-interface/scripts/copyBuiltFiles.ts [hunk 2]
```diff
@@ -31,6 +32,12 @@ async function copyBuiltFiles() {
     copyFile(file, path.resolve(DIST_PATH, file.replace(basePath + '/', './')))
   }
 
+  const webGLStreamingAssetsPath = path.resolve(process.env.BUILD_PATH!, 'StreamingAssets/aa/WebGL')
```
Comment (menduz) [inline browser-interface/scripts/copyBuiltFiles.ts:35]: why is it /aa/?

Comment (dalkia) [reply browser-interface/scripts/copyBuiltFiles.ts:35]: Its the default `LocalBuild` path by Unity. We could remove the aa if we wanted to
image

**Review by Kinerius — APPROVED** [review]

**Pull request merged**